test/decorators/project_decorator_test.rb

```diff
@@ -4,6 +4,14 @@
 
 class ProjectDecoratorTest < ActiveSupport::TestCase
   let(:linux) { create(:project) }
+  before do
+    admin = create(:admin)
+    stubs(:current_user).returns(admin)
```

Comment:
This is equivalent to writing `current_user = admin`. It doesn't stub the actual `current_user` in ProjectDecorator. Remove this line and use `admin` instead of `current_user` in the specs.